core/src/main/scala/com/criteo/cuttle/Executor.scala

```diff
@@ -203,6 +210,23 @@ case class Execution[S <: Scheduling](
     hasBeenCancelled
   }
 
+  def forceSuccess()(implicit user: User): Boolean = {
+    val shouldForceSuccess = atomic { implicit txn =>
+      if (forcedSuccess()) {
+        false
+      } else {
+        forcedSuccess() = true
+        true
+      }
+    }
+    if (shouldForceSuccess) {
```

Comment:
It isn't necessary, just check focedSucces

Reply:
Removed it